lib/Controller/SAMLController.php

```diff
@@ -316,23 +316,37 @@ public function assertionConsumerService() {
 	 * @throws Error
```

Comment:
I Think we need `@PublicPage` as well

Else if you log out of Nextcloud and the SLO gets called for your user and thus you get redirected back to Nextcloud you won't be logged in. right?

Reply:
Not sure this case can append but it makes sense to avoid a login request when a non logged user want to be disconnected :). Committed.